htdocs/core/lib/pdf.lib.php

```diff
@@ -1553,6 +1553,7 @@ function pdf_getlinedesc($object, $i, $outputlangs, $hideref = 0, $hidedesc = 0,
 
 	// Now we convert \n into br
 	if (dol_textishtml($libelleproduitservice)) {
+		$libelleproduitservice = str_replace("\n", '<br/>', $libelleproduitservice);
```

Comment:
If your text $libelleproduitservice  is a html text, the \n into it is not a new line, but something that must not appears on PDF.
If $libelleproduitservice has a HTML format, and need a new line into content, the <br> must be inside the content $libelleproduitservice.  You can't have some part of var that is html and some that are not. All must be non html or all must use the html syntax.

Reply:
Not sure...

**First Example :**
1°) Create a product with a multilined description
![image](https://user-images.githubusercontent.com/82267780/141314781-9e153581-eb2b-44d8-a95e-53f73d77cfa3.png)

The description use \n

2°) Create a propal for example with this product and generate the pdf
![image](https://user-images.githubusercontent.com/82267780/141315054-d1cbecbd-176d-4bd7-a3e9-98fac9290a33.png)

Its'ok because the is no HTML tag

3°) Then set the variable PDF_BOLD_PRODUCT_LABEL and generate again the PDF

![image](https://user-images.githubusercontent.com/82267780/141315323-a93c916a-d603-47d6-bb32-1e0d7422aefc.png)

The result is bad, because Product label is bolded width <b></b> but product description still use \n

**The second one :**
Set the variable SHOW_SUBPRODUCT_REF_IN_PDF
The result may change if you bold or not a line ... bescause in the pdf lib use \n in the code

![image](https://user-images.githubusercontent.com/82267780/141316025-99f2e8d8-f4e5-403d-90c4-4353ab82fcff.png)

Same things with the variable MAIN_DOCUMENTS_DESCRIPTION_FIRST. The result is different if you bold something or not



The other way i see to solve this problem is to replace all valid \n char to __N__ 
For exempla you have 

![image](https://user-images.githubusercontent.com/82267780/141317026-a72a17de-c477-4b8b-be78-af04f074eabf.png)


$desc must be checked.... Because it may use valid \n

![image](https://user-images.githubusercontent.com/82267780/141317227-1b209c28-a6b0-4060-b3b8-f8e1ed22b9ee.png)

The same kind of things must be done in some other lines of the code

Reply:
I have updated code to force valid \n to be replaced by __N__